lighthouse-core/report/v2/renderer/details-renderer.js

```diff
@@ -190,6 +194,7 @@ if (typeof module !== 'undefined' && module.exports) {
 /**
  * @typedef {{
  *     type: string,
+ *     styleAs: (string|undefined),
```

Comment:
since this is so table specific, it probably makes sense to make a new `TableItemJSON` or something